<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpenConnection</class>
 <widget class="QDialog" name="DialogOpenConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>217</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>217</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>217</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open Connection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>9</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>35</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>61</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>61</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>87</y>
     <width>186</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Will open in a new window instead of the current one.</string>
   </property>
   <property name="text">
    <string>Open connection in a new window</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bookmark it</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Bookmarks:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>140</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoConnectCheckbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Will connect to the selected bookmark when the application is launched.</string>
   </property>
   <property name="text">
    <string>Auto-connect to bookmark on launch</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>DialogOpenConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
